File: java/src/Solution557.java
public class Solution557 {
    public String reverseWords(String s) {
        String[] strings = s.split(" ");
        StringBuilder result = new StringBuilder();
        for (int i = 0; i < strings.length; ++i) {
            strings[i] = new StringBuffer(strings[i]).reverse().toString();
            result.append(strings[i]);
            if (i != strings.length - 1) {
                result.append(" ");
            }
        }
        return result.toString();
    }
}

File: python/Solution557.py
class Solution:
    def reverseWords(self, s):
        word_list = s.split(" ")
        for i in range(len(word_list)):
            word_list[i] = word_list[i][::-1]
        result = ""
        for i in range(len(word_list)):
            result += word_list[i]
            if i != len(word_list) - 1:
                result += " "
        return result


if __name__ == '__main__':
    result = Solution().reverseWords("take LeetCode contest")
    print(result)
